python/glow/gwas/lin_reg.py

```diff
@@ -191,6 +187,13 @@ def _linear_regression_inner(genotype_pdf: pd.DataFrame, Y_state: YState,
     '''
 
     X = np.column_stack(genotype_pdf[_VALUES_COLUMN_NAME].array)
+    del genotype_pdf[_VALUES_COLUMN_NAME]
+    num_genotypes = genotype_pdf.shape[0]
+    out_df = pd.concat([genotype_pdf] * Y_state.Y.shape[1])
+    if verbose_output:
+        out_df["n"] = list(np.ravel(Y_mask.T @ np.ones(X.shape)))
```

Comment:
nit: I think this would be both faster and easier to understand as `Y_mask.sum(axis=1).repeat(num_genotypes)` or something along those lines

Reply:
Thanks Henry, I've added this optimization